src/text/__tests__/Tag.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, cleanup } from "@testing-library/react"
+import Tag from "../Tag"
+
+afterEach(cleanup)
+
+describe("<Tag>", () => {
+  it("displays the tag", () => {
+    const content = "Tag text here"
+    const { getByText, container } = render(<Tag id="test-tag" className="test-class">{content}</Tag>)
+    expect(getByText(content)).toBeInTheDocument()
+    expect(container.querySelector(".seeds-tag")).not.toBeNull()
+    expect(container.querySelector("#test-tag")).not.toBeNull()
+    expect(container.querySelector("#test-tag.test-class")).not.toBeNull()
+  })
+
+  it("supports variants", () => {
```

Comment:
I'd recommend removing this test and the test below, they both are a little too close to testing implementation details IMO